src/main/java/com/xiaomi/infra/pegasus/client/ClientOptions.java

```diff
@@ -213,6 +221,18 @@ public Builder falconPushInterval(Duration falconPushInterval) {
       return this;
     }
 
+    /**
+     * The delay time of enable backup request. Defaults to {@literal -1}, which means there is no
+     * need to send backup request, see {@link #DEFAULT_BACKUP_REQUEST_DELAY_MS}.
+     *
+     * @param backupRequestDelayMS backupRequestDelayMS
+     * @return {@code this}
+     */
+    public Builder backupRequestDelayMS(int backupRequestDelayMS) {
```

Comment:
`backupRequestDelayMS` should be a table configuration rather than a client configuration. Because some tables may have strict latency requirements, but some may not.